web/pages/speakers/[slug].tsx

```diff
@@ -1,31 +1,33 @@
 import type { GetStaticPaths, GetStaticProps } from 'next';
 import urlJoin from 'proper-url-join';
 import { groq } from 'next-sanity';
-import type { Person } from '../../types/Person';
-import type { Slug } from '../../types/Slug';
 import MetaTags from '../../components/MetaTags';
 import Nav from '../../components/Nav';
 import GridWrapper from '../../components/GridWrapper';
 import Footer from '../../components/Footer';
-import client from '../../lib/sanity.server';
-import { mainEventId } from '../../util/constants';
 import TextBlock from '../../components/TextBlock';
 import ConferenceUpdatesForm from '../../components/ConferenceUpdatesForm';
 import Card from '../../components/Card';
 import HighlightedSpeakerBlock from '../../components/HighlightedSpeakerBlock';
 import SessionCard from '../../components/SessionCard';
 import twitterLogo from '../../images/twitter_logo_black.svg';
 import linkedinLogo from '../../images/linkedin_logo_black.svg';
-import { SPEAKER } from '../../util/queries';
+import client from '../../lib/sanity.server';
+import type { Person } from '../../types/Person';
+import type { Slug } from '../../types/Slug';
+import type { Session } from '../../types/Session';
+import { mainEventId } from '../../util/constants';
+import { sessionStart } from '../../util/session';
+import { PRIMARY_NAV, SPEAKER } from '../../util/queries';
 import styles from '../app.module.css';
 import speakerStyles from './speakers.module.css';
-import { sessionStart } from '../../util/session';
-import { Session } from '../../types/Session';
+import { PrimaryNavItem } from '../../types/PrimaryNavItem';
```

Comment:
Nit: Can you share the convention you've adopted for ordering imports, so we can cut down on the diffs related to this going forward? (Assuming that eslint isn't configured to check it, which it doesn't seem to be.)

And is it a mistake that this type import isn't grouped along with the others further up (Person, Slug etc)?

Reply:
I decided to address this properly in #137.